src/text-field-components.css.ts

```diff
@@ -0,0 +1,118 @@
+import {style} from '@vanilla-extract/css';
+import {sprinkles} from './sprinkles.css';
+import * as mq from './media-queries.css';
+
+export const LABEL_LEFT_POSITION = 12;
+export const DEFAULT_WIDTH = 328;
+
+// to scale to the correct text-preset when the transition applies
+export const LABEL_SCALE_DESKTOP = parseFloat('0.78'); // Text1/Text3 = 14/18 (desktop)
+export const LABEL_SCALE_MOBILE = parseFloat('0.75'); // Text1/Text3 = 12/16 (mobile)
```

Comment:
why parseFloat?

`0.78 === parseFloat('0.78')`

Reply:
lol, no idea, I didn't notice that xD